bin/near

```diff
@@ -146,52 +153,16 @@ const keys = {
   handler: (argv) => exitOnError(main.keys(argv))
 };
 
-const txStatus = {
-  command: 'tx-status <hash>',
-  desc: 'lookup transaction status by hash',
-  builder: (yargs) => yargs
-    .option('hash', {
-      desc: 'base58-encoded hash',
-      type: 'string',
-      required: true
-    }),
-  handler: (argv) => exitOnError(main.txStatus(argv))
-};
-
-const clean = {
-  command: 'clean',
-  desc: 'clean the build environment',
-  builder: (yargs) => yargs
-    .option('outDir', {
-      desc: 'build directory',
-      type: 'string',
-      default: './out'
-    }),
-  handler: (argv) => exitOnError(main.clean(argv))
-};
-
-const newProject = {
-  command: 'new_project [projectDir]',
-  desc: 'create a new blank project',
-  builder: (yargs) => yargs
-    .option('projectDir', {
-      desc: 'project directory',
-      type: 'string',
-      default: '.'
-    }),
-  handler: (argv) => exitOnError(main.newProject(argv))
-};
-
 const stake = {
-  command: 'stake [accountId] [publicKey] [amount]',
+  command: 'stake <accountId> <stakeKey> <amount>',
```

Comment:
`stakingKey` makes more sense to me

Reply:
is staking a word that means the process not the result? the command is stake should it really be staking make more sense?